worlds/sc2/locations.py

```diff
@@ -753,6 +755,12 @@ def get_locations(world: Optional['SC2World']) -> Tuple[LocationData, ...]:
                      lambda state: logic.zerg_common_unit(state) and logic.zerg_basic_anti_air(state)),
         make_location_data(SC2Mission.SHOOT_THE_MESSENGER.mission_name, "East Frozen Zerg", SC2HOTS_LOC_ID_OFFSET + 509, LocationType.EXTRA,
                      lambda state: logic.zerg_common_unit(state) and logic.zerg_competent_anti_air(state)),
+        make_location_data(SC2Mission.SHOOT_THE_MESSENGER.mission_name, "West Launch Bay", SC2HOTS_LOC_ID_OFFSET + 510, LocationType.CHALLENGE,
+                     lambda state: logic.zerg_common_unit(state) and logic.zerg_competent_anti_air(state)),
+        make_location_data(SC2Mission.SHOOT_THE_MESSENGER.mission_name, "Center Launch Bay", SC2HOTS_LOC_ID_OFFSET + 511, LocationType.CHALLENGE,
+                     lambda state: logic.zerg_common_unit(state) and logic.zerg_competent_anti_air(state)),
+        make_location_data(SC2Mission.SHOOT_THE_MESSENGER.mission_name, "East Launch Bay", SC2HOTS_LOC_ID_OFFSET + 512, LocationType.CHALLENGE,
+                     lambda state: logic.zerg_common_unit(state) and logic.zerg_competent_anti_air(state)),

```

Comment:
Are unupgraded hydras OK?

Reply:
Personally I believe so, yes. People often claim that this is a very easy alternative win condition so I'm inclined to assume it doesn't need a comp army. However, I'm open to changing that if there is a need.

Reply:
the thing is if "only" unupgraded hydras will make it (no roaches or lings), no Kerrigan abilities

Reply:
On discord we just had a discussion and decided `zerg_competent_comp` would probably be best. Changing it to that.